### PR TITLE
Fixes #10345: On the Rule details, if I click on "Clone", then cancel action, I get switched to Settings tab

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10345

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/components/popup/CreateRulePopup.scala [hunk 1]
```diff
@@ -183,9 +183,11 @@ class CreateOrCloneRulePopup(
   private[this] def error(msg:String) = <span class="col-lg-12 errors-container">{msg}</span>
 
   private[this] def closePopup() : JsCmd = {
-      JsRaw("""
-        $('#createRulePopup').bsModal('hide');
-        $('#settingsTab').bsTab('show');
+      JsRaw(s"""
+        $$('#createRulePopup').bsModal('hide');
+        if(!${clonedRule.isDefined}){
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/components/popup/CreateRulePopup.scala]: I think we should just remove the bs tab. 

The problem her eis that the tab changed even we did not want

Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/components/popup/CreateRulePopup.scala]: @RaphaelGauthier ?

Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/components/popup/CreateRulePopup.scala]: @RaphaelGauthier ?

**Comment by fanf** [comment]
This PR was replaced by ttps://github.com/Normation/rudder/pull/1590

**Pull request closed without merging**